### PR TITLE
undo summary capital letter in Run Summary

### PR DESCRIPTION
**Pull request by trilopin**
#### Summary
In playbook_edit, "Run Summary" copy is shown.
Replace it with "Run summary" and use correct capitalization.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-42857

#### Checklist
- [ ] ~~Telemetry updated~~
- [ ] ~~Gated by experimental feature flag~~
- [ ] ~~Unit tests updated~~

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -67,6 +67,13 @@
     "id": "app.user.digest.tasks.heading",
     "translation": "Your assigned tasks"
   },
+  {
```
Comment (lieut-data) [inline assets/i18n/en.json:70]: This change surprises me -- is there some non-determinism between platforms that explains why the tool "moved" this?

Comment (trilopin) [reply assets/i18n/en.json:70]: Yeah, I noticed that, and also surprised me.

Comment (lieut-data) [reply assets/i18n/en.json:70]: I'll defer to @calebroseland since I /think/ he has the most context here.

**Review by calebroseland — APPROVED** [review]

File: assets/i18n/en.json (hunk 1 above)
Comment (calebroseland) [reply assets/i18n/en.json:70]: (I'll follow up on this separately—will go ahead and merge)

**Pull request merged**